corporate/tests/test_stripe.py

```diff
@@ -589,24 +599,111 @@ def test_upgrade_by_invoice(self, *mocks: Mock) -> None:
                 'Billed by invoice']:
             self.assert_in_response(substring, response)
 
+    def test_upgrade_page(self) -> None:
+        user = self.example_user("hamlet")
+        self.login_user(user)
+
+        response = self.client_get("/upgrade/")
+        self.assert_in_success_response([
+            'Upgrade to Zulip Standard',
+            'Pay automatically',
+            'Pay by invoice',
+            'Payment schedule',
+            'License management',
+            'Request sponsorship',
+            'Organization type',
+        ], response)
+        self.assert_not_in_success_response([
+            'Choose a plan for your new organization',
+            '60 day free trial',
```

Comment:
` day free trial` seems better to check - "60 day free trial" wouldn't be found in the response even if things were bugged relative to what we expect and e.g. there was "30 day free trial" in the response

Reply:
Good catch. Fixed.